docs/monorepo.md

```diff
@@ -38,8 +38,30 @@ test/
 
 Your `package.json` can have any of the [normal properties](https://docs.npmjs.com/files/package.json) but at a minimum should contain `main`, `module`, and `sideEffects`.
 
+### devDependencies
+
 It used to be that `devDependencies` needed to be added to the root `package.json` but since we moved to `yarn` workspaces we're able to add them as regular `devDependencies` within the package that uses them.
 
+### dependencies
+
+Running the following in your `wp-calypso` root <br/>
+`yarn workspace @automattic/{{your-package}} run prepare && npx @yarnpkg/doctor packages/{{your-package}}` <br/>
```

Comment:
is `<br />` a valid/proper markdown to use here? I think it's better just to apply line breaks.

Reply:
I see only two `<br>`s in the whole doc, just fyi.

Reply:
Great observation! Actually I am using a markdown previewer and it looked good, but I searched it a little more and....
TIL: to add a line break in .md add two or more spaces and then hit return (one space & return won't work).
https://github.com/Automattic/wp-calypso/pull/44481/commits/f22d36be22c8067fa74b35c0878056044a48b6fa

Thanks @retrofox !